src/SelectMenu/SelectMenuItem.tsx

```diff
@@ -65,7 +65,6 @@ export const listItemStyles = css`
 
   // can hover states
   @media (hover: hover) {
-    body:not(.intent-mouse) .SelectMenu-item:focus,
```

Comment:
I can't find any instances of the `.SelectMenu-item` class being used.  Possible I overlooked it, but I think this can be removed at this point.